docs/how-to-use-gvfs.md

```diff
@@ -0,0 +1,225 @@
+| `fs.gravitino.server.uri`                             | The Gravitino server uri which gvfs needs to load the fileset meta.                                                                                                                               | (none)        | Yes      | 0.5.0         |
+| `fs.gravitino.client.metalake`                        | The metalake which fileset belongs.                                                                                                                                                               | (none)        | Yes      | 0.5.0         |
+| `fs.gravitino.fileset.cache.maxCapacity`              | The cache capacity in the Gravitino Virtual File System.                                                                                                                                          | `20`          | No       | 0.5.0         |
+| `fs.gravitino.fileset.cache.evictionMillsAfterAccess` | The value of time that the cache evicts the element after access in the Gravitino Virtual File System. The value is in `milliseconds`.                                                            | `300000`      | No       | 0.5.0         |
+
+You can configure these properties in two ways:
+
+1. Before getting the `FileSystem` in the code, construct the `Configuration` and set the properties:
+
+    ```java
+    Configuration conf = new Configuration();
+    conf.set("fs.AbstractFileSystem.gvfs.impl","com.datastrato.gravitino.filesystem.hadoop3.Gvfs");
+    conf.set("fs.gvfs.impl","com.datastrato.gravitino.filesystem.hadoop3.GravitinoVirtualFileSystem");
+    conf.set("fs.gravitino.server.uri","http://localhost:8090");
+    conf.set("fs.gravitino.client.metalake","test_metalake");
+    Path filesetPath = new Path("gvfs://fileset/test_catalog/test_schema/test_fileset_1");
+    FileSystem fs = filesetPath.getFileSystem(conf);
+    ```
+
+2. Configure these properties in the `core-site.xml` file of the Hadoop environment:
+
+    ```xml
+      <property>
+        <name>fs.AbstractFileSystem.gvfs.impl</name>
+        <value>com.datastrato.gravitino.filesystem.hadoop3.Gvfs</value>
+      </property>
+
+      <property>
+        <name>fs.gvfs.impl</name>
+        <value>com.datastrato.gravitino.filesystem.hadoop3.GravitinoVirtualFileSystem</value>
+      </property>
+
+      <property>
+        <name>fs.gravitino.server.uri</name>
+        <value>http://localhost:8090</value>
+      </property>
+
+      <property>
+        <name>fs.gravitino.client.metalake</name>
+        <value>test_metalake</value>
+      </property>
+    ```
+
+## How to use
+
+Please make sure to have the Gravitino Virtual File System runtime jar firstly, you can get it in
+two ways:
+
+1. [Maven repository](https://mvnrepository.com/): to download the runtime jar in the Maven central
```

Comment:
Download from the maven central repo: You can get the package which name is like `gravitino-filesystem-hadoop3-runtime-{version}.jar` from the [Maven repository](https://mvnrepository.com/).